app/screens/settings/Settings.tsx

```diff
@@ -13,6 +13,8 @@ import { smColors } from '../../vars';
 import { Account, AppThDispatch, RootState } from '../../types';
 import { Modal } from '../../components/common';
 
+import { version as appVersion } from '../../../package.json';
```

Comment:
It seems that `package.json` infects the codebase.
Don't you think that is much better to store it once in reducer (as `currentVerson` id addition to the default value for `latestVersion`) and then use it in Components via selector?

Reply:
I'll rewrite it into the app reducer now.

Reply:
Fixed.

Reply:
Resolved, on initial load, `currentVersion` and `latestVersion` are set to the package.json version. `latestVersion` is updated if new version is available.